zh-TW.i18n.json

```diff
@@ -0,0 +1,263 @@
+    "archive": "刪除",
+    "archive-all": "全部刪除",
+    "archive-board": "刪除看板",
+    "archive-card": "刪除卡片",
+    "archive-list": "刪除該清單",
+    "archive-selection": "刪除所選內容",
+    "archiveBoardPopup-title": "確定要刪除看板嗎？",
+    "archived-items": "回收筒",
+    "archives": "回收筒",
+    "assign-member": "分配成員",
+    "attached": "附加",
+    "attachment": "附件",
+    "attachment-delete-pop": "刪除附件的操作無法還原。",
+    "attachmentDeletePopup-title": "刪除附件？",
+    "attachments": "附件",
+    "avatar-too-big": "大頭貼太大 (最大 70 Kb)",
+    "back": "返回",
+    "board-change-color": "更改顏色",
+    "board-nb-stars": "%s 星號標記",
+    "board-not-found": "看板不存在",
+    "board-private-info": "該看板將被設為 <strong>私有</strong>.",
+    "board-public-info": "該看板將被設為 <strong>公開</strong>.",
+    "boardChangeColorPopup-title": "修改看板背景",
+    "boardChangeTitlePopup-title": "重新命名看板",
+    "boardChangeVisibilityPopup-title": "更改可視級別",
+    "boardImportBoardPopup-title": "從 Trello 匯入看板",
+    "boardMenuPopup-title": "看板選單",
+    "boards": "看板",
+    "bucket-example": "例如 “目標清單”",
+    "cancel": "取消",
+    "card-archived": "該卡片已被刪除",
+    "card-comments-title": "該卡片有 %s 則評論",
+    "card-delete-notice": "徹底刪除的操作不可復原，你將會遺失該卡片相關的所有操作記錄。",
+    "card-delete-pop": "所有的動作將從活動動態中被移除且您將無法重新打開該卡片。此操作無法復原。",
+    "card-delete-suggest-archive": "你可以將卡片從看板中刪除至回收筒，但保留相關活動。",
+    "card-edit-attachments": "編輯附件",
+    "card-edit-labels": "編輯標籤",
+    "card-edit-members": "編輯成員",
+    "card-labels-title": "更改該卡片上的標籤",
+    "card-members-title": "在該卡片中新增或移除看板成員",
+    "cardAttachmentsPopup-title": "附件來源",
+    "cardDeletePopup-title": "徹底刪除卡片？",
+    "cardDetailsActionsPopup-title": "卡片動作",
+    "cardLabelsPopup-title": "標籤",
+    "cardMembersPopup-title": "成員",
+    "cardMorePopup-title": "更多",
+    "cards": "卡片",
+    "change": "變更",
+    "change-avatar": "更改大頭貼",
+    "change-password": "更改密碼",
+    "change-permissions": "更改許可權",
+    "changeAvatarPopup-title": "更改大頭貼",
+    "changeLanguagePopup-title": "更改語言",
+    "changePasswordPopup-title": "更改密碼",
+    "changePermissionsPopup-title": "更改權限",
+    "click-to-star": "點此來標記該看板",
+    "click-to-unstar": "點此來去除該看板的標記",
+    "clipboard": "剪貼簿貼上或者拖曳檔案",
+    "close": "關閉",
+    "close-board": "關閉看板",
+    "close-board-pop": "您可以透過點選主介面上方的”回收筒“按鈕來還原看板。",
+    "color-black": "黑色",
+    "color-blue": "藍色",
+    "color-green": "綠色",
+    "color-lime": "綠黃",
+    "color-orange": "橙色",
+    "color-pink": "粉紅",
+    "color-purple": "紫色",
+    "color-red": "紅色",
+    "color-sky": "天藍",
+    "color-yellow": "黃色",
+    "comment": "評論",
+    "comment-placeholder": "新增評論",
+    "computer": "從本機上傳",
+    "create": "建立",
+    "createBoardPopup-title": "建立看板",
+    "createLabelPopup-title": "建立標籤",
+    "current": "目前",
+    "decline": "拒絕",
+    "default-avatar": "預設大頭貼",
+    "delete": "刪除",
+    "deleteLabelPopup-title": "刪除標籤？",
+    "description": "描述",
+    "disambiguateMultiLabelPopup-title": "清除標籤動作歧義",
+    "disambiguateMultiMemberPopup-title": 清除成員動作歧義",
```

Comment:
For the record, I fixed this small syntax error before pushing this json file to Transifex.